async-nats/src/jetstream/stream.rs

```diff
@@ -169,6 +169,11 @@ pub struct StreamInfo {
     pub state: StreamState,
 }
 
+#[derive(Deserialize)]
+pub struct DeleteResponse {
```

Comment:
Something other than response, maybe status?

```suggestion
pub struct DeleteStatus {
```

Reply:
Good point.